frontend/app/src/scenes/Home/scenes/Ambassadors/index.tsx

```diff
@@ -24,41 +35,61 @@ const columns = [
     dataIndex: 'firstName',
     key: 'firstName',
     // eslint-disable-next-line react/display-name
-    render: (firstName: string) => <Title level={4}>{firstName}</Title>,
+    render: (firstName: string) => <span>{firstName}</span>,
   },
   {
     title: 'Last Name',
     dataIndex: 'lastName',
     key: 'lastName',
     // eslint-disable-next-line react/display-name
-    render: (lastName: string) => <Title level={4}>{lastName}</Title>,
+    render: (lastName: string) => <span>{lastName}</span>,
```

Comment:
If you don't add any css classes here no need a Span tag.

also if you won't process the data you can just remove the `render` key.
It will look like this:
```
{
    title: 'Last Name',
    dataIndex: 'lastName',
    key: 'lastName',
}
```

Reply:
ok